infra/aws/terraform/registry-sandbox-k8s-io-image-layers/s3-origin-access-logs.tf

```diff
@@ -0,0 +1,77 @@
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+
+resource "aws_s3_bucket" "access_log" {
+  provider = aws.origin
+
+  bucket        = "${aws_s3_bucket.origin.bucket}-access-log"
+  force_destroy = false
+
+  depends_on = [
+    aws_s3_bucket.origin
+  ]
+}
+
+resource "aws_s3_bucket_acl" "access_log" {
+  provider = aws.origin
+
+  bucket = aws_s3_bucket.access_log.id
+  acl    = "log-delivery-write"
+}
+
+resource "aws_s3_bucket_server_side_encryption_configuration" "access_log" {
+  provider = aws.origin
+
+  bucket = aws_s3_bucket.access_log.id
+
+  rule {
+    apply_server_side_encryption_by_default {
+      sse_algorithm = "AES256"
+    }
+  }
+}
+
+resource "aws_s3_bucket_lifecycle_configuration" "access_log" {
+  provider = aws.origin
+
+  bucket = aws_s3_bucket.access_log.id
+
+  rule {
+    id     = "auto-delete"
+    status = "Enabled"
+
+    filter {}
+
+    # Objects are deleted after 90 days
+    expiration {
+      days = 90
```

Comment:
nit: I'd put the number of days into a `local`

Reply:
Done